style.go

```diff
@@ -524,7 +524,9 @@ func pad(str string, n int, style *ansi.Style) string {
 		return str
 	}
 
-	sp := strings.Repeat(" ", abs(n))
+	// We use a non-breaking space to pad so that the padding is
```

Comment:
this make sense but we are sure we want this for all the cases?

no-break space is used to introduce a space where you don't want line breaks, like where it'll look strange or easily be misunderstood if a line break.

Reply:
That's why we want them here in padding.

Reply:
ah right, sorry had to ask just to double check